deploy/k8s-osc-ccm/values.yaml

```diff
@@ -25,4 +25,13 @@ podLabels: {}
 # -- Assign Pod to Nodes (see [kubernetes doc](https://kubernetes.io/docs/tasks/configure-pod-container/assign-pods-nodes/))
 nodeSelector: {}
 # -- Pod tolerations (see [kubernetes doc](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/))
-tolerations: []
\ No newline at end of file
+tolerations: []
+
+# -- HTTP proxy configuration example 
+# caBundle:
+#   name: my-ca-bundle
+#   key: my-ca-bundle.pem
+
+# httpsProxy: https://my-proxy.foo.bar
+
+noProxy: 10.0.0.0/8,172.16.0.0/12,192.168.0.0/16,169.254.169.254
```

Comment:
Can you also comment it ?

Reply:
Done in commit 3032aee6a3428b064b6ef1cca4c1323123bcafa2